crates/biome_js_parser/tests/js_test_suite/error/import_type_error.ts

```diff
@@ -0,0 +1,4 @@
+type A = typeof import("a.json",{});
```

Comment:
Can we create a new file, with a case where we have EOF? e.g. 

```ts
type A = typeof import("a.json",{
```

Reply:
ok, of course